test/ngResource/resourceSpec.js

```diff
@@ -978,6 +979,194 @@ describe("basic usage", function() {
       });
     });
 
+
+    describe('requestInterceptor', function() {
+      var rejectReason = {'lol':'cat'};
+      var testError = new Error();
```

Comment:
This is not used. And I miss the following: _Test that throwing inside the `requestInterceptor` calls the `requestErrorInterceptor`._
Coincidence ? :stuck_out_tongue: